View/ViewHandler.php

```diff
@@ -430,7 +431,7 @@ public function createResponse(View $view, Request $request, $format)
     {
         $route = $view->getRoute();
         $location = $route
-            ? $this->getRouter()->generate($route, (array) $view->getRouteParameters(), true)
+            ? $this->getRouter()->generate($route, (array) $view->getRouteParameters(), UrlGeneratorInterface::ABSOLUTE_URL)
```

Comment:
this fixes an issue since in newer versions one actually needs to pass ``0`` to get absolute urls